packages/regl-worldview/src/commands/Cones.js

```diff
@@ -6,19 +6,17 @@
 //  found in the LICENSE file in the root directory of this source tree.
 //  You may not use this file except in compliance with the License.
 
+import * as React from "react";
 import type { BaseShape } from "../types";
 import fromGeometry from "../utils/fromGeometry";
-import { getObjectFromHitmapId, getHitmapProps } from "../utils/hitmapDefaults";
-import { makeCommand } from "./Command";
+import { createInstancedGetHitmap } from "../utils/getHitmapDefaults";
+import Command, { type OptionalCommandProps } from "./Command";
 import { createCylinderGeometry } from "./Cylinders";
 
 const { points, sideFaces, endCapFaces } = createCylinderGeometry(30, true);
 
 const cones = fromGeometry(points, sideFaces.concat(endCapFaces));
 
-const Cylinders = makeCommand<BaseShape>("Cylinders", cones, {
-  getHitmapProps,
-  getObjectFromHitmapId,
-});
-
-export default Cylinders;
+export default function Cones(props: { ...OptionalCommandProps, children: Array<BaseShape> }) {
+  return <Command getHitmap={createInstancedGetHitmap({ pointCountPerInstance: 1 })} {...props} reglCommand={cones} />;
```

Comment:
nit: ordering
```suggestion
  return <Command reglCommand={cones} getHitmap={createInstancedGetHitmap({ pointCountPerInstance: 1 })} {...props} />;
```

Reply:
We don't want users of this command to overwrite the reglCommand ever though, do we? I thought that was the one prop we don't want to allow

Reply:
Ah gotcha, yeah that makes sense. I think I suggested this because it matches the order of props in other places we use Command. Not a big deal